tests/infrastructure/numa/test_basic_numa.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+
+from tests.infrastructure.numa.utils import assert_qos_guaranteed
+from tests.utils import (
+    assert_numa_cpu_allocation,
+    assert_virt_launcher_pod_cpu_manager_node_selector,
+    get_numa_node_cpu_dict,
+    get_vm_cpu_list,
+)
+from utilities.virt import migrate_vm_and_verify, validate_libvirt_persistent_domain
+
+
+class TestBasicCx1Numa:
+    @pytest.mark.polarion("CNV-12364")
+    def test_numa_pod_resource_limits_match_requests(self, created_vm_cx1_instancetype):
+        container = created_vm_cx1_instancetype.vmi.virt_launcher_pod.instance.spec.containers[0]
+        limits = container.resources.limits or {}
+        requests = container.resources.requests or {}
+
+        mismatches = []
+        for key in ("cpu", "memory", "hugepages-1Gi"):
```

Comment:
are these the only keys that exist under limits and requests? if so you can `requests == limits`

Reply:
there are other keys too but their behavior doesn't impact us . Added the comment back which i accidentally removed